src/app/ubs/ubs-admin/components/ubs-admin-table/table-cell-readonly/table-cell-readonly.component.scss

```diff
@@ -5,6 +5,7 @@ div {
   overflow: hidden;
   text-overflow: ellipsis;
   padding: 0 20px;
+  min-width: 160px !important;
```

Comment:
Add class to needed div and write min-width there because use !important is a bad practice